### ISSUE TITLE
ICICI bank home page !  the first element of the home page graphic logo link is not label properly

### ISSUE THREAD

**Issue by nsingh2001** [nsingh2001/bicici#2]
UAT 2022-04
Windows 10
Chrome Version 100.0.4896.88 (Official Build) (64-bit)

NVDA 2021.3.1
Repro Steps
1. Open the URL  https://www.icicibank.com/
2. The first element of the home page ‘ graphic logo link
 
Expected Result the first element of the page graphic logo should be announce like icici bank logo link

Actual Result ‘graphic logo link’ announced by screen reader 
WCAG Guideline
1. https://www.w3.org/TR/WCAG21/#name-role-value
